mne/preprocessing/xdawn.py

```diff
@@ -62,6 +69,12 @@ def _least_square_evoked(epochs_data, events, tmin, sfreq):
         An concatenated array of toeplitz matrix for each event type.
     """
 
+    if isinstance(solver, string_types):
+        if solver == "pinv":
+            from ..stats.regression import _pinv_solver as solver
+        elif solver == 'cholesky':  # noqa
```

Comment:
I meant: I don't get what lines are counted as not being checked/why I get the drop in coverage.